libs/core/src/lib/file-uploader/file-uploader.component.scss

```diff
@@ -1,2 +1,6 @@
 @import '~fundamental-styles/dist/input';
 @import '~fundamental-styles/dist/file-uploader';
+
+.fd-input.is-error {
+    border-color: var(--sapField_InvalidColor, #b00) !important;
```

Comment:
I am not sure if we can add this styling, that too with `!important` here. Can this not be done through platform code itself, since Core may not have requirement for state handling?

Reply:
can this not be added in platform instead through `fdp-file-uploader .fd-file-uploader .fd-input.is-error {` . I am still not sure we need `important` since we have specificity now.